TAO/TAO_IDL/util/utl_global.cpp

```diff
@@ -1965,11 +1965,7 @@ namespace
         idl_global->set_lineno (1);
         idl_global->set_filename (0);
 
-        // Name this pseudo-file "builtin-N"
-        static char buffer[64];
-        static unsigned n = 1;
-        ACE_OS::snprintf (&buffer[0], sizeof buffer, "builtin-%u", n++);
-        UTL_String utl_string (&buffer[0], true);
+        UTL_String utl_string = get_filename();
```

Comment:
```suggestion
        UTL_String utl_string = this->get_filename ();
```

Reply:
Ok, so I can assumed that using "this->" is the preferred "style" in ACE/TAO?

Reply:
https://github.com/DOCGroup/ACE_TAO/blob/9097c5600873fd147884c0903fc285f1be0c2de7/ACE/docs/ACE-guidelines.html#L806